feature-group-definitions/sanitizer.yml

```diff
@@ -1 +1,2 @@
+name: "Sanitizer API"
```

Comment:
Hmm, I don't think this needs "API". The API entrypoint is the `Sanitizer` constructor.

```suggestion
name: "Sanitizer"
```